rcsb/db/tests/testPdbCsmImageSplitter.py

```diff
@@ -0,0 +1,138 @@
+##
+# File:    testPdbCsmImageWorkflow.py
+# Author:  Michael Trumbull
+# Date:    12-Dec-2024
+# Version: 0.01
+#
+# Updates:
+#
+##
+
+__docformat__ = "google en"
+__author__ = "Michael Trumbull"
+__email__ = "michael.trumbull@rcsb.org"
+__license__ = "Apache 2.0"
+__version__ = "V0.01"
```

Comment:
Can you delete this? We don't usually keep track of specific versions of individual files, just the package.